pkg/kv/kvserver/testdata/echotest/TestMultiSSTWriterInitSST

```diff
@@ -0,0 +1,29 @@
+echo
+----
+>> sst0:
+rangedel: /Local/RangeID/0/{r""-s""}
+rangekeydel: /Local/RangeID/0/r"{"-\x00"}
+rangekeyset: /Local/RangeID/0/{r"\x00"-s""} -> /<empty>
+rangekeydel: /Local/RangeID/0/{r"\x00"-s""}
+>> sst1:
+rangedel: /Local/Range{"d"-/Max}
+rangekeyset: /Local/RangeID/0/s"{"-\x00"} -> /<empty>
+rangekeydel: /Local/Range"d"{-}
+rangekeyset: /Local/Range{"d"-/Max} -> /<empty>
+rangekeydel: /Local/Range{"d"-/Max}
+>> sst2:
+rangedel: /Local/Lock/Local/Range{"d"-/Max}
+rangekeyset: /Local/Range/Max{-} -> /<empty>
+rangekeydel: /Local/Lock/{Local/Range"d"-""\x12\x01k\x12d\x00\xff\x01\x00\x01\x00""}
+rangekeyset: /Local/Lock/{""\x12\x01k\x12d\x00\xff\x01\x00\x01\x00""-Local/Range/Max} -> /<empty>
+rangekeydel: /Local/Lock/{""\x12\x01k\x12d\x00\xff\x01\x00\x01\x00""-Local/Range/Max}
+>> sst3:
+rangedel: /Local/Lock{"d"-/Max}
+rangekeyset: /Local/Lock/{Local/Range/Max-""\x12\x01k\x12\xff\xff\x00\xff\x01\x00\x01\x00""} -> /<empty>
+rangekeydel: /Local/Lock{"d"-/""\x12d\x00\x01\x00""}
+rangekeyset: /Local/Lock/{""\x12d\x00\x01\x00""-Max} -> /<empty>
+rangekeydel: /Local/Lock/{""\x12d\x00\x01\x00""-Max}
+>> sst4:
+rangedel: {d-/Max}
+rangekeyset: /Local/Lock/{Max-""\x12\xff\xff\x00\x01\x00""} -> /<empty>
+rangekeyset: {d\x00-\xff\xff\x00} -> /<empty>
```

Comment:
This seems fishy. Why the `\x00` on both start and end key?

Reply:
Oh, in this case these are always just prefixes, but all valid prefixes end in \x00 (they are also valid keys).

Reply:
There was a bug here :-) I doubly encoded keys in some case (see [here](https://docs.google.com/document/d/1uvyFWuJg4FWl2OVwg3ekQlVNBb5CjCj_muUr3HG-oKU/edit?tab=t.0))

I fixed it now, the \x00 are gone, and CI passes. 😅

Reply:
Oh, sorry, then this must print the roach key not the encoded prefix (i.e. roach key + \x00).